crates/rspack/tests/tree-shaking/ts-target-es5/expected/main.js

```diff
@@ -4,17 +4,9 @@
 Object.defineProperty(exports, "__esModule", {
     value: true
 });
-function _export(target, all) {
-    for(var name in all)Object.defineProperty(target, name, {
-        enumerable: true,
-        get: all[name]
-    });
-}
-_export(exports, {
-    _async_to_generator: function() {
-        return _async_to_generator;
-    },
-    _: function() {
+Object.defineProperty(exports, "_", {
```

Comment:
This is expected, since only `_` is used.
[_async_to_generator](https://github.com/web-infra-dev/rspack/pull/3547/files#diff-7778d5149babbc138f4afc0e9aaf627ebed05f11856a32c6fd4c118890fa893cR219)